static/js/components/CommentForms.js

```diff
@@ -65,59 +66,77 @@ const commentForm = (
   isComment: boolean,
   disabled: boolean,
   autoFocus: boolean,
-  wysiwyg: boolean = false
-) => (
-  <div className="reply-form">
-    <form
-      onSubmit={onSubmit}
-      className="form"
-      onKeyDown={e => {
-        if (e.key === "Enter" && e.ctrlKey && !disabled && !isEmptyText(text)) {
-          onSubmit(e)
+  wysiwyg: boolean = false,
+  onTogglePopup?: Function
+) => {
+  return (
```

Comment:
can we keep this as a one-line arrow function (like `() => (<div />)` rather than

```js
() => {
  return (
    <div />
  )
}
```
?

I think the one-line version is just a little cleaner, and it will mean fewer spurious changes showing up in the diff